docs/getting_started.md

```
@@ -14,38 +14,45 @@ This tutorial will provide a brief overview of:
 ## Configuring the Library
 
-Right-click on the Project datafeed (*i.e.*, **Definitions**) and select **Properties**. Navigate to the **Landfire** tab, where you can specify details about the quantity, type, and composition of the vegetation for your model.
+Right-click on the Project datafeed (*i.e.*, **BpSModels**) and select **Open** (or **double-click** on BpSModels). Navigate to the **LANDFIRE vegetation models** tab, where you can specify details about the quantity, type, and composition of the vegetation for your model.
+
+<img align="middle" style="padding: 1px" width="500" src="assets/images/image-4.png">
 
-<img align="middle" style="padding: 1px" width="500" src="assets/images/project-landfire-tab-opened.png">
+Open the Scenario datafeed (*i.e.*, the LANDFIRE BpS Reference Condition Models). Navigate to the **LANDFIRE vegetation models** tab, where you can provide further information about the model, including detailed **Succession Class Descriptions** and **Succession Class Mapping Rules**.
 
-Right-click on the Scenario datafeed (*i.e.*, the Scenario name) and select **Properties**. Navigate to the **Landfire** tab, where you can provide further information about the model, including detailed **Succession Class Descriptions** and **Succession Class Mapping Rules**.
+<img align="middle" style="padding: 1px" width="500" src="assets/images/image-5">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)